ocrd_cis/ocropy/recognize.py

```diff
@@ -107,20 +100,20 @@ def get_model(self):
         be resolved with OcrdResourceManager to a valid readeable file and
         returns it.  If not, it checks if the model can be found in the
         dirname(__file__)/models/ directory."""
-        canread = lambda p: os.path.isfile(p) and os.access(p, os.R_OK)
+        canread = lambda p: isfile(p) and access(p, R_OK)
         try:
             model = self.resolve_resource(self.parameter['model'])
             if canread(model):
                 return model
         except SystemExit:
-            ocropydir = os.path.dirname(os.path.abspath(__file__))
-            path = os.path.join(ocropydir, 'models', self.parameter['model'])
+            ocropydir = dirname(abspath(__file__))
+            path = join(ocropydir, 'models', self.parameter['model'])
             self.logger.info("Failed to resolve model with OCR-D/core mechanism, trying %s", path)
             if canread(path):
                 return path
         self.logger.error("Could not find model %s. Try 'ocrd resmgr download ocrd-cis-ocropy-recognize %s",
                 self.parameter['model'], self.parameter['model'])
-        sys.exit(1)
+        exit(1)
 
     def process(self):
```

Comment:
and here

Reply:
Check [19566c0](https://github.com/bertsky/ocrd_cis/pull/5/commits/19566c0567b5b23bdc4596384d3867601045ca57) and [5f60976](https://github.com/bertsky/ocrd_cis/pull/5/commits/5f60976452011656fd05c1375055dd5ebd5f89d9). If good, I will remove the `process()` method.